test/Grpc.StatusProto.Tests/ExceptionExtensionsTest.cs

```diff
@@ -0,0 +1,100 @@
+#region Copyright notice and license
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using System.Text;
+using Google.Rpc;
+using NUnit.Framework;
+
+namespace Grpc.StatusProto.Tests;
+
+/// <summary>
+/// Tests for ExceptionExtensions
+/// </summary>
+public class ExceptionExtensionsTest
+{
+    [Test]
+    public void ToRpcDebugInfoTest()
+    {
+        try
+        {
+            ThrowException("extra details");
```

Comment:
You can use `var ex = Assert.Throws<Exception>(() => ThrowException("extra details"));` instead of try/catch. It's a bit more idiomatic way of writing tests that deal with exceptions.

For example:

https://github.com/grpc/grpc-dotnet/blob/246c43d4574a5e2ba3fb4f397df304dfb5688b2d/test/Grpc.AspNetCore.Server.Tests/ServerCallContextExtensionsTests.cs#L44